iml-orm/src/lib.rs

```diff
@@ -167,6 +167,12 @@ mod change {
         iter::FromIterator,
     };
 
+    pub trait Identifiable {
+        type Id: Eq + Ord;
```

Comment:
Maybe require `Debug` as well?

Reply:
I'm only adding the required traits for now. We don't use `Debug` or `Hash` when calling `Identifiable` below. If we end up needing them later we can change this restriction at the same time.